omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -5,577 +5,727 @@
+        }
 
-                /* enable viral load copies field */
-                jq("#viral-load-results-qualitative").change(function () {
-                    var viral_load_status = getValue('viral-load-results-qualitative.value');
-                    if (viral_load_status == "") {
-                        jq("#viral-load-status").hide();
-                    }
-
-                    if (viral_load_status == 1301) {
-                        /* Detected */
-                        <ifMode mode="ENTER">
-                            setValue('viral-load-results-copies.value', '');
-                        </ifMode>
-
-                        var viral_load_copies = getValue('viral-load-results-copies.value');
-                        if(viral_load_copies == 1) {
-                            jq("#viral-load-status").hide();
-                            setValue('viral-load-results-copies.value', '');
-                        }
-                    }
-                    if (viral_load_status == 1306) {
-                        /* Not Detected */
-                        jq("#viral-load-status").hide();
-                        setValue('viral-load-results-copies.value', 1);
-                    }
-                });
+        if (currentRegimen !== jq("#90315").find("select").find(":selected").val()) {
+        jq("#99025 input").val(0);
+        } else {
+        jq("#99025 input").val(durationOnCurrentRegimen);
+        }
+        DSDMStabilityCheck();
+        });
+        /* End of Validation */
+
+        jq("#90203").change(function () {
+        clinicStaging = jq(this).find(":selected").val();
+        DSDMStabilityCheck();
+        });
+
+        disable_fields("164975");
+        jq("#160288").change(function () {
+        if (jq(this).find("select").find(":selected").val() == "90002") {
+        enable_fields("164975");
+        } else {
+        jq("#164975").find("input[type=hidden]").val() == " ";
+        disable_fields("164975");
+        }
+        });
+
+        disable_fields("5596");
+        disable_fields("90012");
+        disable_fields("162983");
+        disable_fields("anc_no");
+
+        /* enable the edd,emtct,age in months and ANC no. fields in case the client is pregnant */
+
+        jq(".pregnancy").change(function () {
+        if (jq(this).find('#90041_1065').find(':checkbox').prop('checked')) {
+        enable_fields("5596");
+        enable_fields("90012");
+        enable_fields("162983");
+        enable_fields("anc_no");
+        enable_fields("162983");
+        } else {
+        disable_fields("5596");
+        disable_fields("90012");
+        disable_fields("162983");
+        disable_fields("anc_no");
+        disable_fields("162983");
+        }
+        });
+
+        jq(".pregnancy").change(function () {
```

Comment:
@slubwama This is an example of JS that needs to be indented

Reply:
I believe It is indented now

Reply:
done